src/liblibc/lib.rs

```diff
@@ -1931,10 +1933,30 @@ pub mod types {
                 pub type PLONG = *mut c_long;
 
                 #[cfg(target_arch = "x86")]
+                pub type LONGLONG = c_double;
```

Comment:
The msdn documentation is full of lies. Don't trust it. It is only defined as `double` for really old versions of msvc which did not have compiler support for 64-bit integers. Nowadays it is defined as `typedef unsigned __int64 ULONGLONG;` even for 32-bit.